### PR TITLE
aus der Asche

### PR DESCRIPTION
**Pull request by MMMiracles**
ok the turf change made in awhile ago and I finally found the will to try and finish this after dealing with issues from both git and real life lets go kids.

Map is converted and compiles, although a bit out-dated compared to what I had locally before the turf change was pulled. PR is up to show I'm not actually dead yet and have interest in finishing this. 

weeeeeeeee

### CODE REVIEW

**Comment by Zilenan91** [comment]
The hero has done it

**Comment by PKPenguin321** [comment]
THE HERO

**Comment by kevinz000** [comment]
DO I SEE
AN AWAY MISSION?!!!!!!

**Comment by Zilenan91** [comment]
It's a space nazi moon base away mission

**Comment by duncathan** [comment]
THE TRUE HERO OF OUR TIME

File: code/modules/awaymissions/mission_code/raisearm.dm [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+	retreat_distance = 5
+	minimum_distance = 5
+	icon_state = "nsoldierranged"
+	icon_living = "nsoldierranged"
+	casingtype = /obj/item/ammo_casing/c45nostamina
+	projectilesound = 'sound/weapons/Gunshot_smg.ogg'
+
+/mob/living/simple_animal/hostile/nsoldier/melee //hulk nazi with really buff biceps for added effect
+	name = "Hulking Geo-Nazi"
+	desc = "The result severe genetic testing and steriods. Would probably win an arm-wrestling match."
```
Comment (JJRcop) [inline code/modules/awaymissions/mission_code/raisearm.dm]: of

**Comment by optimumtact** [comment]
Literally HITLER :de:

File: code/modules/awaymissions/mission_code/raisearm.dm [hunk 2]
```diff
@@ -0,0 +1,457 @@
...
+	..()
+	if(iscarbon(target))
+		var/mob/living/carbon/C = target
+		if(prob(90))
+			C.Weaken(2)
+			src.say("HEIL HITLER!")
+			explosion(src, 0, 0, 2, 3, 2)
+			src.gib()
+
+/mob/living/simple_animal/hostile/spawner/heilfactory
```
Comment (Dorsisdwarf) [inline code/modules/awaymissions/mission_code/raisearm.dm]: hilariously-named, but should be nazifactory for ease of being found

File: code/modules/awaymissions/mission_code/raisearm.dm [hunk 3]
```diff
@@ -0,0 +1,457 @@
...
+	retreat_distance = 3
+	minimum_distance = 2
+	faction = list("german")
+	projectilesound = 'sound/weapons/Gunshot_smg.ogg'
+	deathmessage = "Mecha Hitler's body activates its self-destruct function!"
+	loot = list(/obj/effect/gibspawner/robot)
+	wanted_objects = list()
+	search_objects = 0
+	spawn_mecha_type = /obj/mecha/combat/marauder/mauler/roboh
+
```
Comment (Dorsisdwarf) [inline code/modules/awaymissions/mission_code/raisearm.dm]: on_death should make it explode catastrophically

File: code/modules/awaymissions/mission_code/raisearm.dm [hunk 4]
```diff
@@ -0,0 +1,457 @@
...
+	desc = "A banner depicting a rather rude image. Has a strange, smokey smell to it."
+	icon_state = "heil"
+
+/obj/structure/banner/cyka
+	name = "russian banner"
+	desc = "A banner. Smells faintly of vodka."
+	icon_state = "cyka"
+
+/obj/structure/banner/assblastusa
+	name = "patriotic banner"
```
Comment (Supermichael777) [inline code/modules/awaymissions/mission_code/raisearm.dm]: American banner you ethnocentric fool

**Comment by GunHog** [comment]
No pictures?

File: code/modules/awaymissions/mission_code/raisearm.dm (hunk 1 above)
Comment (octareenroon91) [inline code/modules/awaymissions/mission_code/raisearm.dm]: steroids*

**Comment by JJRcop** [comment]
:red_circle::white_circle::black_circle:

**Comment by Cheridan** [comment]
seriously though, you can't add swasticas and hitler to the game

**Comment by Zilenan91** [comment]
these are space swastikas and space hitlers though it's totally different

**Comment by MMMiracles** [comment]
why can't i add buddhism peace signs and historically inaccurate german soldiers. wolfienstien did it just fine.

**Comment by optimumtact** [comment]
Yeah cheridan what the hell?

**Comment by Dorsisdwarf** [comment]
@KorPhaeron We gave you executive design control, and now you must make a decision.

Can we have some swastiks up in this nazi base?

**Comment by KorPhaeron** [comment]
Cheridan outranks me sorry

**Comment by MMMiracles** [comment]
What if I rename them "historically inaccurate soldier of German ethnicity" and change the symbols into Buddhism peace

**Comment by Jordie0608** [comment]
Change them into east-german communists, they looked almost exactly the same, sans swastika

**Comment by Zilenan91** [comment]
Why do they need to be changed though? None of the charm that comes with Space Nazi Moon Base will carry over which is basically the entire appeal of it

**Comment by Dorsisdwarf** [comment]
Replace the swastikas with Luftwaffe crosses to avoid "wah nazi"
The secret is they're re-enactors who've been terribly mislead

**Comment by clusterfack** [comment]
Wow Wolfenstein 2D, we'll need to port this

**Comment by MMMiracles** [comment]
k i made it more PC so now SS13 won't get banned in Germany

**Comment by Probe1** [comment]
Awesome.

**Comment by MMMiracles** [comment]
to be quite honest im not sure about anything anymore and i just cant cant FUCK

**Comment by demonfiren** [comment]
I did Nazi that coming.

Tell me you're Goering to keep working on this.

**Comment by MMMiracles** [comment]
jk

**Comment by MMMiracles** [comment]
ok so i ran into another roadblock in the form of the map and all backups of it are crashing in saving (nothing else does). different versions of byond and outreach at the irc had no results.

eeeeee

**Comment by KorPhaeron** [comment]
@MMMiracles 

What is the story on this?

**Comment by MMMiracles** [comment]
dream maker keeps crashing when trying to save the map itself, i've tried backups that I had locally/uploaded, changing byond versions, and starting a new branch entirely. 

seeing how away missions are pretty much non-supported anymore with lavaland taking place for the exploring, at most I'll probably pull some stuff from here and drop it in as lavaland ruins but other than that, I think I'm done with this.

**Comment by PKPenguin321** [comment]
Fuck

Rest in peace, nazi mission. You were just too fun for this world.

**Comment by KorPhaeron** [comment]
Shame we never got to play it in full

**Comment by Dorsisdwarf** [comment]
Add the nazis to the code

**Pull request closed without merging**